cmd/personality.go

```diff
@@ -22,3 +22,6 @@ var DisplayName string = "Pebble"
 
 // DefaultDir is the Pebble directory used if $PEBBLE is not set.
 var DefaultDir string = "/var/lib/pebble/default"
+
+// StateFile is the file name of the state file in pebble dir.
+var StateFile string = ".pebble.state"
```

Comment:
I guess having a separate `StateFile` variable is better than using `"."+ProgramName+".state"`?

Reply:
Yes, it'd allow us to be more flexible with the file name selection.